data/mapcss-mapping.csv

```diff
@@ -70,7 +70,8 @@ railway|rail|main;[railway=rail][usage=main][!highspeed][!service];;name;int_nam
 landuse|residential;15;
 highway|path;16;
 highway|bus_stop;17;
-natural|tree;18;
+# ~4M usages.
+highway|footway|sidewalk;[highway=footway][footway=sidewalk];;name;int_name;18;
```

Comment:
Hm, some thoughts:
- do we need natural=tree in future say for 3D?
- I think we will add nodes highway=crossing (8M) soon

Keep these _good_ values reserved?

Reply:
Let's postpone trees for the future. They not only pollute the map visually, but also slow down mwm reading a bit. 

Trees were displayed in MapsWithMe, but we disabled them to avoid user confusion on the map.